lib/arjdbc/jdbc/arel_support.rb

```diff
@@ -1,3 +1,31 @@
+if Arel::VERSION >= "7.1.0"
```

Comment:
It was removed here as unused https://github.com/rails/arel/commit/62e3596db89570b9d48b13c9966d754d5859d0dc

I didn't find any alternative yet